tuxemon/core/db.py

```diff
@@ -192,3 +192,6 @@ def set_defaults(results, table):
                 )
 
     return results
+
+# Global database container
+databases = JSONDatabase()
```

Comment:
`databases` is misleading.  this is a single object.  please change to `database`, or simply `db`

Reply:
Renamed to db.